### PR TITLE
[1LP][RFR]Tests ssui/services tests cleanup

### PR DESCRIPTION
**Pull request by niyazRedhat**
Purpose or Intent
=================
1. Reassigning assignee
2. Added casecomponent
3. Added tags

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit ce7a098fa3a83a0194b27283ac556bdb6032e76f:

`cfme/tests/automate/test_service_dialog.py`:
- :heavy_exclamation_mark: Line 245:1: **W293** *blank line contains whitespace*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/ssui/test_ssui_dashboard.py [hunk 1]
```diff
@@ -135,8 +135,10 @@ def test_total_services(appliance, setup_provider, context, order_service):
     """Tests total services count displayed on dashboard.
 
     Polarion:
-        assignee: sshveta
+        assignee: nansari
+        casecomponent: SelfServiceUI
         initialEstimate: 1/4h
+        tags: service
```
Comment (lina-nikiforova) [inline cfme/tests/ssui/test_ssui_dashboard.py]: Q: why do you have different tags for ssui tests? Here it's `service`, later it's `ssui`

**Comment by lina-nikiforova** [comment]
Q: why you didn't reassign `test_sui_monitor_ansible_playbook_std_output` and `test_sui_snapshots_for_vm_create_edit_delete`?

Otherwise looks good to me.

File: cfme/tests/ssui/test_ssui_dashboard.py (hunk 1 above)
Comment (niyazRedhat) [reply cfme/tests/ssui/test_ssui_dashboard.py]: thanks for pointing.making changes

**Comment by niyazRedhat** [comment]
@lina-nikiforova reassigned my name on those test cases

**Pull request merged**